yarn-project/circuits.js/src/structs/l2_to_l1_message.ts

```diff
@@ -0,0 +1,68 @@
+import { EthAddress } from '@aztec/foundation/eth-address';
+import { Fr } from '@aztec/foundation/fields';
+import { BufferReader, FieldReader, serializeToBuffer } from '@aztec/foundation/serialize';
+
+export class L2ToL1Message {
```

Comment:
As earlier, this one can be really weird to follow as we have the same name for solidity and outside, but it is not talking about the same thing.

Reply:
"Addressed" this just by adding there comments in https://github.com/AztecProtocol/aztec-packages/commit/6d5c87b72f8e545bed662319544c118eea037ddc